models/Document/PageSnippet.php

```diff
@@ -70,6 +71,11 @@ abstract class PageSnippet extends Model\Document
      */
     protected $contentMasterDocumentId;
 
+    /**
+     * @var null|int
+     */
+    protected $requireEditableValues = null;
```

Comment:
I'd prefer `hasRequiredEditables` as the name for this property, as it indicates whether this document has at least one editable configured as required or not. 
@weisswurstkanone what do you think?

Reply:
In other places we're also using e.g. `validateRequiredEditables` so I guess this would be more consistent.

Reply:
@brusch that's not true, actually it states that if a document is missing values for any required editables.

Reply:
@dvesh3 oh dear, my fault, sorry 😊
But I'm still not entirely happy with the name, what do you think about `missingRequiredEditable` or `hasMissingRequiredEditable`?

Reply:
@brusch I think `missingRequiredEditable` fits better ;)